testutil/ciao-down/prepare.go

```diff
@@ -260,8 +260,66 @@ func finishedFN(ws *workspace) string {
 	return fmt.Sprintf(finishedStr, ws.HTTPServerPort)
 }
 
+func proxyVarsFN(ws *workspace) string {
+	var buf bytes.Buffer
+	if ws.NoProxy != "" {
+		buf.WriteString("no_proxy=")
+		buf.WriteString(ws.NoProxy)
+		buf.WriteString(" ")
```

Comment:
Are you sure that the software that expects HTTP_PROXY (i.e. capitalised) doesn't also expect NO_PROXY too?